packages/epics/src/membership/components/inner-spaces/inner-spaces-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import { InnerSpaceCard } from './inner-space-card';
+import { useInnerSpaces } from '../../hooks/use-inner-spaces';
+
+type InnerSpacesListProps = {
+  page: number;
+  activeSort: string;
+};
+
+export const InnerSpacesList: FC<InnerSpacesListProps> = ({
+  page,
+  activeSort,
+}) => {
+  const { innerSpaces, isLoading } = useInnerSpaces({
+    page,
+    sort:
+      activeSort === 'all'
+        ? { sort: 'all' }
+        : activeSort === 'most-recent'
+        ? { sort: 'most-recent' }
+        : undefined,
```

Comment:
This signature `{sort: {sort: ...}}` is very confusing. The complexity is caused by the type mismatch `SortParams` vs `string` for `activeSort`.
```